app/controllers/api/v1/confirmations_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class ConfirmationsController < DeviseTokenAuth::ConfirmationsController
+      def show
+        self.resource = resource_class.confirm_by_token(params[:confirmation_token])
+        yield resource if block_given?
+
+        if resource.errors.empty?
+          set_flash_message!(:notice, :confirmed)
+          render json: { 'message': 'user has been confirmed' }, status: :accepted
```

Comment:
Is a good practice to have this strings in a `locales` file.

See https://guides.rubyonrails.org/i18n.html